Duplicate variables in types (Cannot redeclare block-scoped variable 'daysInWeek')

Seems that in master the **typings.d.ts** has duplicate declared variables for **daysInWeek** (Line 1197)

`error TS2451: Cannot redeclare block-scoped variable 'daysInWeek'.

1195   const daysInWeek: number`

https://github.com/date-fns/date-fns/blob/master/typings.d.ts#L1197


![duplicate-variable](https://user-images.githubusercontent.com/87658684/180245027-d7a5d43d-ac4a-4b3a-a941-291e7dbaa6df.png)

Was introduced by the commit https://github.com/date-fns/date-fns/commit/6d8e424021f31edacc46b8b55a823fa3b54cc08c

--- a/src/constants/index.d.ts
+++ b/src/constants/index.d.ts
@@ -1,7 +1,7 @@
 // This file is generated automatically by `scripts/build/typings.js`. Please, don't change it.
 
 export const daysInWeek: number
-export const daysInWeek: number
+export const daysInYear: number
 export const maxTime: number
 export const millisecondsInMinute: number
 export const millisecondsInHour: number

--- a/src/constants/index.ts
+++ b/src/constants/index.ts
@@ -14,9 +14,11 @@ export const daysInWeek = 7
  *
  * > Leap year occures every 4 years, except for years that are divisable by 100 and not divisable by 400.
  * > 1 mean year = (365+1/4-1/100+1/400) days = 365.2425 days
- *  @constant
- *  @type {number}
- *  @default
+ *
+ * @name daysInYear
+ * @constant
+ * @type {number}
+ * @default
  */
 export const daysInYear: number = 365.2425
 

--- a/src/fp/index.js.flow
+++ b/src/fp/index.js.flow
@@ -674,7 +674,7 @@ declare module.exports: {
   yearsToMonths: CurriedFn1<number, number>,
   yearsToQuarters: CurriedFn1<number, number>,
   daysInWeek: number,
-  daysInWeek: number,
+  daysInYear: number,
   maxTime: number,
   millisecondsInMinute: number,
   millisecondsInHour: number,

--- a/src/index.js.flow
+++ b/src/index.js.flow
@@ -890,7 +890,7 @@ declare module.exports: {
 
   daysInWeek: number,
 
-  daysInWeek: number,
+  daysInYear: number,
 
   maxTime: number,
 

--- a/typings.d.ts
+++ b/typings.d.ts
@@ -1196,7 +1196,7 @@ declare module 'date-fns' {
 
   const daysInWeek: number
 
-  const daysInWeek: number
+  const daysInYear: number
 
   const maxTime: number
 
@@ -1233,7 +1233,7 @@ declare module 'date-fns' {
 
 declare module 'date-fns/constants' {
   export const daysInWeek: number
-  export const daysInWeek: number
+  export const daysInYear: number
   export const maxTime: number
   export const millisecondsInMinute: number
   export const millisecondsInHour: number
@@ -1254,7 +1254,7 @@ declare module 'date-fns/constants' {
 
 declare module 'date-fns/constants/index' {
   export const daysInWeek: number
-  export const daysInWeek: number
+  export const daysInYear: number
   export const maxTime: number
   export const millisecondsInMinute: number
   export const millisecondsInHour: number
@@ -1275,7 +1275,7 @@ declare module 'date-fns/constants/index' {
 
 declare module 'date-fns/constants/index.js' {
   export const daysInWeek: number
-  export const daysInWeek: number
+  export const daysInYear: number
   export const maxTime: number
   export const millisecondsInMinute: number
   export const millisecondsInHour: number
@@ -6021,7 +6021,7 @@ declare module 'date-fns/fp' {
 
   const daysInWeek: number
 
-  const daysInWeek: number
+  const daysInYear: number
 
   const maxTime: number
 
@@ -10972,7 +10972,7 @@ declare module 'date-fns/esm' {
 
   const daysInWeek: number
 
-  const daysInWeek: number
+  const daysInYear: number
 
   const maxTime: number
 
@@ -15734,7 +15734,7 @@ declare module 'date-fns/esm/fp' {
 
   const daysInWeek: number
 
-  const daysInWeek: number
+  const daysInYear: number
 
   const maxTime: number
 
@@ -23766,7 +23766,7 @@ interface dateFns {
 
   daysInWeek: number
 
-  daysInWeek: number
+  daysInYear: number
 
   maxTime: number
 

